[DOCS-6531] MwL Tag Exclusion Updates (#20320)
DataDog/documentation: content/en/metrics/summary.md

* **Configure Tags**: Configure tags on multiple custom metrics matching a namespace using Metrics without Limits™

> {{< img src="metrics/summary/bulkconfig_new.mp4" alt="Bulk Metric Tag Configuration" video=true style="width:100%;"}} 

Reviewer: ```suggestion
{{< img src="metrics/summary/bulkconfig_new.mp4" alt="Bulk Metric Tag Configuration" video=true style="width:100%;">}} 
```

1. Click on your custom distribution metric name in the **Metrics Summary** table to open the metrics details side panel.
2. Click the **Manage Tags** button to open the tag configuration modal.

> 3. There are two ways in which you can configure your tags:

Reviewer: Duplicating instructions and information across multiple pages can lead to maintenance issues in the future. How about redirecting customers to the more in-depth instructions you list in the Metrics without Limits page?
```suggestion

For more information on tag configuration, see the [Metrics without Limits][X] documentation.
```
Add the [X] link at the bottom of the page

Author: That makes sense! Thank you for pointing that out :)

1. Click on your custom distribution metric name in the **Metrics Summary** table to open the metrics details side panel.
2. Click the **Manage Tags** button to open the tag configuration modal.

> For more information on tag configuration, see the [Metrics without Limits][X] documentation.

Reviewer: ```suggestion
3. Select **Include tags...** or **Exclude tags...** to customize the tags you do or don't want to query for. For more information on tag configuration, see the [Metrics without Limits][10] documentation.
```

>   - Click the **Include tags...** tab to customize the tags you'd like to keep available in an _allowlist_ of the tags you'd like to keep.

Reviewer: ```suggestion
```

>   - Click the **Exclude tags...** tab to customize the tags you don't want to query for in an _blocklist_ of tags you want to drop.

Reviewer: ```suggestion
```
